fusionSuite/frontend
# New/types

Changes proposed in this pull request:

- Add types management + import templates page
- couple fixes in display

How to test the feature manually:

1. have menu *Types* and after test all

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._

Related to #
